UnityMirror/UnityServer/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -144,6 +144,11 @@ public static void Start()
 
         public static void ReadyForPlayers()
         {
+            if(CurrentState.CurrentGameState == GameState.Active)
```

Comment:
Is this what we have in the other GSDK? Just wondering if we change it to say only allowed to call when it is initializing or not set, but this should be ok in general

Reply:
it's a bit tricky since in the other GSDKs we block, unfortunately we can't block in Unity. We've had that in the Unity GSDK source code for a while (on the GSDK repo https://github.com/PlayFab/gsdk/blob/master/UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs#L146), we can adjust if necessary.